### PR TITLE
Enhance/single store page header menu and search filter responsiveness

### PR DESCRIPTION
**Pull request by tanjiralmamun**
### All Submissions:

* [x] My code follows the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [ ] My code has proper inline documentation
* [ ] I've included developer documentation (optional)
* [ ] I've added proper labels to this pull request

### Changes proposed in this Pull Request:

Single Store Page store header menu and search fields style break on mobile devices. This PR has fixed the issue.

Screenshots:
- Previous - Desktop: https://snipboard.io/JGabY0.jpg
- Previous - Mobile: https://snipboard.io/TKzwg2.jpg
- Proposed Changes - Desktop: https://snipboard.io/RnKJAM.jpg
- Proposed Changes - Mobile: https://snipboard.io/TDcEub.jpg

Closes#.

### How to test the changes in this Pull Request:

1.
2.
3.

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -4483,6 +4483,22 @@ div.media-sidebar a.edit-attachment {
   background-size: 25px 25px;
   z-index: 999;
 }
+@media (max-width: 480px){
```
Comment (nurul-umbhiya) [inline assets/css/style.css:4486]: Style.css changes will be overwritten after running the grunt command.
Move responsive related css code to response.less and store related css to store.less

Comment (nurul-umbhiya) [reply assets/css/style.css:4486]: Also exclude this file from this pr

Comment (tanjiralmamun) [reply assets/css/style.css]: Done Bhai

**Comment by tanjiralmamun** [comment]
> Scenario: design break for lower resolution
> 
> Screenshot: sf sf1

Fixed has been committed.

**Pull request merged**